### DISCUSSION TITLE
DotNet.Testcontainers.Containers.ResourceReaperException : Initialization has been cancelled.

### DISCUSSION THREAD

**Discussion by tjpeel** [testcontainers/testcontainers-dotnet#903] [Q&A]
I have seen the lengthy discussion on this topic but sadly it's affecting me on our Buildkite CI server.

I have a dotnet test project in Xunit and I'm starting a Kafka instance and a custom Aerospike instance. When running in Rider everything works OK, when running locally via Docker (as in the tests are running within their own container) it works OK too.

On the build server the tests run within Docker.

Local env:

[[testcontainers.org](http://testcontainers.org/) 00:00:00.09] Connected to Docker:
  Host: unix:///var/run/docker.sock
  Server Version: 20.10.23
  Kernel Version: 5.15.49-linuxkit
  API Version: 1.41
  Operating System: Docker Desktop
  Total Memory: 9.73 GB

CI build agent env:

[[testcontainers.org](http://testcontainers.org/) 00:00:00.06] Connected to Docker:
  Host: unix:///var/run/docker.sock
  Server Version: 19.03.3
  Kernel Version: 3.10.0-1160.80.1.el7.x86_64
  API Version: 1.40
  Operating System: CentOS Linux 7 (Core)
  Total Memory: 125.75 GB

I am using latest testcontainers version 3.1.0.

What are my next steps to debugging the issue further?

Many thanks

**Comment by HofmeisterAn** [comment]
How do you install Docker on Buildkite? Can you try to run and connect to [moby-ryuk](https://github.com/testcontainers/moby-ryuk) manual

    docker run -d -v /var/run/docker.sock:/var/run/docker.sock -e RYUK_PORT=8080 -p 8080:8080 testcontainers/ryuk:0.5.1
    nc localhost 8080

and get the container logs afterward?

**Reply by tjpeel** [reply]
I'll have a look and see what I can do, thanks for the reply

**Reply by tjpeel** [reply]
I have paused this work for the time being hence the delay. Hopefully I can get back to it soon
